src/spritekit.cs

```diff
@@ -1954,28 +1991,28 @@ partial interface SKAction : NSCoding, NSCopying {
 		
 		// These are in a category
 		[Static, Export ("moveByX:y:duration:")]
-		SKAction MoveBy (nfloat deltaX, nfloat deltaY, double sec);
+		SKAction MoveBy (nfloat deltaX, nfloat deltaY, double duration);
```

Comment:
@spouliot is this a breaking change? I am in favor of the change if we can allow it but @VincentDondain  I would use `durationInSeconds` instead if seconds is the unit of measure of duration